contracts/scripts/torii.sh

```diff
@@ -7,7 +7,7 @@ export STARKNET_RPC_URL=http://katana:5050
 
 sozo migrate --rpc-url $STARKNET_RPC_URL
 
-torii --world $SOZO_WORLD --rpc $STARKNET_RPC_URL &
+torii --world $SOZO_WORLD --rpc $STARKNET_RPC_URL --database $WORLD_DB &
```

Comment:
we configure the WORLD_DB from the docker_compose file

we have torii act as the primary, i.e. where the world_db is 'synced from' and then the loremachine is just a replica, `synced_to`